extensions/eclipse-che-theia-terminal/package.json

```diff
@@ -22,6 +22,7 @@
     "@theia/core": "next",
     "@theia/terminal": "next",
     "xterm": "3.9.2",
+    "reconnecting-websocket": "^4.2.0",
```

Comment:
Do we need CQ for it?

Reply:
Yes, created [here](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21065).